modules/oe_subscriptions_anonymous/tests/src/Functional/TermsConfigurationTest.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_subscriptions_anonymous\Functional;
+
+use Drupal\Core\Url;
+use Drupal\Tests\BrowserTestBase;
+use Drupal\Tests\flag\Traits\FlagCreateTrait;
+
+/**
+ * Tests the subscription configuration.
+ */
+class TermsConfigurationTest extends BrowserTestBase {
+
+  use FlagCreateTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'node',
+    'oe_subscriptions_anonymous',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $defaultTheme = 'stark';
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp(): void {
+    parent::setUp();
+
+    $this->drupalCreateContentType([
+      'type' => 'article',
+      'name' => 'Article',
+    ]);
+
+    $this->drupalCreateContentType([
+      'type' => 'page',
+      'name' => 'Page',
+    ]);
+  }
+
+  /**
+   * Tests the configuration for the terms link in the subscriptions form.
+   */
+  public function testTermsConfiguration(): void {
```

Comment:
testSettingsForm().
We will test any configuration in here.

Reply:
Renamed, added check for the form field